### PR TITLE
Style clean up to match mocks better

### PR DESCRIPTION
**Pull request by ehanna4**
- Fixed some lint errors
- removed 'cancel' button per convo w Joy
- Tried to make create flow match the mocks better
- commented out some create logic that was causing lint issue

### CODE REVIEW

File: src/components/NewGalaxyModal.js [hunk 1]
```diff
@@ -71,14 +72,11 @@ export const NewGalaxyModal = withModalDrawer({ width: 675 })(class NewGalaxyMod
...
 
-
-
     const renderBottomButtons = () => {
       return h(Fragment, [
         div({ style: { display: 'flex', margin: '3rem 0 1rem' } }, [
           div({ style: { flex: 1 } }),
-          this.renderActionButton(),
-          h(ButtonSecondary, { style: { margin: '0 2rem 0 1rem' }, onClick: onDismiss }, 'Cancel')
+          this.renderActionButton()
```
Comment (ehanna4) [inline src/components/NewGalaxyModal.js:79]: removed cancel button

File: src/pages/workspaces/workspace/Notebooks.js [hunk 2]
```diff
@@ -171,7 +171,10 @@ const NotebookCard = ({ namespace, name, updated, metadata, listView, wsName, on
     notebookMenu,
     title,
     div({ style: { flexGrow: 1 } }),
-    locked && h(Clickable, { style: { display: 'flex', paddingRight: '1rem', color: colors.dark(0.75) }, tooltip: `This notebook is currently being edited by ${lockedBy || 'another user'}` }, [icon('lock')]),
+    locked && h(Clickable, {
```
Comment (ehanna4) [inline src/pages/workspaces/workspace/Notebooks.js:174]: This and the next 3 changes are just lint

File: src/components/NewGalaxyModal.js [hunk 3]
```diff
@@ -117,11 +121,11 @@ export const NewGalaxyModal = withModalDrawer({ width: 675 })(class NewGalaxyMod
 
     const renderDefaultCase = () => {
       return h(Fragment, [
-        div({ style: { ...styles.drawerContent, paddingTop: '0.5rem' } }, [
-          div(['Environment consists of an application and cloud compute.']),
+        div(['Environment consists of an application and cloud compute.']),
+        div({ style: { paddingTop: '0.5rem' } }, [
           div({ style: { ...styles.whiteBoxContainer, marginTop: '1rem' } }, [
```
Comment (s-rubenstein) [inline src/components/NewGalaxyModal.js:126]: I think this marginTop is too much, I tried `0.5rem` which looked a little better.

**Review by s-rubenstein — APPROVED** [review]
Found a couple of other style things to look at changing, but it looks good to me as is right now.

One I could comment on, the other is on https://github.com/DataBiosphere/terra-ui/pull/2229/files#diff-211339171d05470c17a4f450691be6c9R139, cost should be bolded.

File: src/components/NewGalaxyModal.js [hunk 4]
```diff
@@ -89,14 +87,20 @@ export const NewGalaxyModal = withModalDrawer({ width: 675 })(class NewGalaxyMod
...
-            ['Creating a cloud environment for Galaxy takes ', span({ style: { fontWeight: 600 } }, ['8-10 minutes.'])]),
-          p({ style: { margin: 0 } }, ['You can navigate away, and we will notify you when it\'s ready. ']),
-          div({ style: { fontSize: 16, fontWeight: 600 } }, ['Continuation cost']),
-          p({ style: { margin: 0 } }, ['Please delete the cloud environment when finished; it will']),
-          p({ style: { margin: 0 } }, ['continue to ', span({ style: { fontWeight: 600 } }, ['incur charges ']), 'if it keeps running.'])
+        div({ style: { marginBottom: '1rem' } }, ['Environment consists of an application and cloud compute.']),
+        div({ style: { ...styles.whiteBoxContainer, backgroundColor: colors.accent(.1), boxShadow: Style.standardShadow } }, [
+          div({ style: { display: 'flex', padding: `0.5rem 0 0`, minHeight: 120 } }, [
+            div({ style: { flex: '1', lineHeight: '1.5rem', minWidth: 0 } }, [
+              span({ style: { marginRight: '1rem' } }, [icon('info-circle', { size: 25, color: colors.accent() })]),
```
Comment (s-rubenstein) [inline src/components/NewGalaxyModal.js:94]: I think the info icon might be too high up?

File: src/components/NewGalaxyModal.js [hunk 5]
```diff
@@ -89,14 +87,20 @@ export const NewGalaxyModal = withModalDrawer({ width: 675 })(class NewGalaxyMod
...
+              span({ style: styles.headerText }, ['Set up duration']),
+              div({ style: { marginLeft: '2.5rem', lineHeight: 1 } },
+                [p(['Creating a cloud environment for Galaxy takes ', span({ style: { fontWeight: 600 } }, ['8-10 minutes.'])]),
+                  p(['You can navigate away, and we will notify you when it\'s ready. ']),
+                  div({ style: styles.headerText }, ['Continuation cost']),
+                  p(['Please delete the cloud environment when finished; it will']),
```
Comment (s-rubenstein) [inline src/components/NewGalaxyModal.js:100]: The line spacing here seems too high.

**Pull request merged**